File: Statefull/sqlite_image_app/lib/model/address.dart
import 'dart:typed_data';

class Address {
  final int? id;
  final String name;
  final String phone;
  final String address;
  final String relation;
  final Uint8List image;

  Address(
    {
      this.id,
      required this.name,
      required this.phone,
      required this.address,
      required this.relation,
      required this.image,


    }


  );


  Address.fromMap(Map<String, dynamic> res)
  : id = res['id'],
    name = res['name'],
    phone = res['phone'],
    address = res['address'],
    relation = res['relation'],
    image = res['image'];

}

File: Swift/SwiftUIWorkSpace/SQLiteImageAddress/SQLiteImageAddress/Model/Address.swift
//
//  Address.swift
//  SQLiteImageAddress
//
//  Created by MACBOOK on 8/8/25.
//

import SwiftUI

struct Address{
    var id: Int
    var name: String
    var phone: String
    var address: String
    var relation: String
    var image: UIImage
    
    init(id: Int, name: String, phone: String, address: String, relation: String, image: UIImage) {
        self.id = id
        self.name = name
        self.phone = phone
        self.address = address
        self.relation = relation
        self.image = image
    }
}
